packages/wrangler/src/dev.ts

```diff
@@ -40,350 +40,343 @@ import type { CfModule, CfWorkerInit } from "./deployment-bundle/worker";
-				type: "boolean",
-				deprecated: true,
-				hidden: true,
-			})
-			.option("minify", {
-				describe: "Minify the script",
-				type: "boolean",
-			})
-			.option("node-compat", {
-				describe: "Enable Node.js compatibility",
-				type: "boolean",
-			})
-			.option("experimental-enable-local-persistence", {
-				describe:
-					"Enable persistence for local mode (deprecated, use --persist)",
-				type: "boolean",
-				deprecated: true,
-				hidden: true,
-			})
-			.option("persist-to", {
-				describe:
-					"Specify directory to use for local persistence (defaults to .wrangler/state)",
-				type: "string",
-				requiresArg: true,
-			})
-			.option("live-reload", {
-				describe:
-					"Auto reload HTML pages when change is detected in local mode",
-				type: "boolean",
-			})
-			.check((argv) => {
-				if (argv["live-reload"] && argv.remote) {
-					throw new UserError(
-						"--live-reload is only supported in local mode. Please just use one of either --remote or --live-reload."
-					);
-				}
-				return true;
-			})
-			.option("inspect", {
-				describe: "Enable dev tools",
-				type: "boolean",
-				deprecated: true,
-				hidden: true,
-			})
-			.option("legacy-env", {
-				type: "boolean",
-				describe: "Use legacy environments",
-				hidden: true,
-			})
-			.option("test-scheduled", {
-				describe: "Test scheduled events by visiting /__scheduled in browser",
-				type: "boolean",
-				default: false,
-			})
-			.option("log-level", {
-				choices: ["debug", "info", "log", "warn", "error", "none"] as const,
-				describe: "Specify logging level",
-				// Yargs requires this to type log-level properly
-				default: "log" as LoggerLevel,
-			})
-			.option("show-interactive-dev-session", {
-				describe:
-					"Show interactive dev session  (defaults to true if the terminal supports interactivity)",
-				type: "boolean",
-			})
-			.option("experimental-dev-env", {
-				alias: ["x-dev-env"],
-				type: "boolean",
-				deprecated: true,
-				hidden: true,
-			})
-			.option("experimental-registry", {
-				alias: ["x-registry"],
-				type: "boolean",
-				describe:
-					"Use the experimental file based dev registry for multi-worker development",
-				default: false,
-			})
-			.option("experimental-vectorize-bind-to-prod", {
-				type: "boolean",
-				describe:
-					"Bind to production Vectorize indexes in local development mode",
-				default: false,
-			})
-	);
-}
-
-type DevArguments = StrictYargsOptionsToInterface<typeof devOptions>;
-
-export async function devHandler(args: DevArguments) {
-	await printWranglerBanner();
+const command = defineCommand({
+	command: "wrangler dev",
+	behaviour: {
+		printConfigWarnings: false,
+	},
+	metadata: {
+		description: "👂 Start a local server for developing your Worker",
+		owner: "Workers: Authoring and Testing",
+		status: "stable",
+	},
+	positionalArgs: ["script"],
+	args: {
+		script: {
+			describe: "The path to an entry point for your worker",
```

Comment:
We have some inconsistency in capitalisation for `Worker` - I feel like we capitalise it in most places.